tox.ini

```diff
@@ -28,8 +34,13 @@ commands = flake8 pythonforandroid/ tests/ ci/
 
 [flake8]
 ignore =
-    E123, E124, E126,
-    E226,
-    E402, E501,
-    W503,
-    W504
+    E123,  # Closing bracket does not match indentation of opening bracket's line
+    E124,  # Closing bracket does not match visual indentation
+    E126,  # Continuation line over-indented for hanging indent
+    E127,  # Continuation line over-indented for visual indent
+    E129,  # Visually indented line with same indent as next logical line
```

Comment:
Nice that you're adding the description for them all :clap: 
My review on this was pending and I forgot to submit.
Basically I was asking not to add E127 and E129 in the exception list since we've been successfully honored it until then.
I agree at first some PE8 feels annoying, but then we get used to and it definitely helps with consistency over a collaborative project like p4a